badger/cmd/backup.go

```diff
@@ -55,7 +55,7 @@ func doBackup(cmd *cobra.Command, args []string) error {
 	opt := badger.DefaultOptions(sstDir).
 		WithValueDir(vlogDir).
 		WithTruncate(truncate).
-		WithNumVersionsToKeep(math.MaxUint32)
+		WithNumVersionsToKeep(int(math.MaxUint32))
```

Comment:
Unless I'm missing something, this doesn't seem necessary.

Reply:
Right, this should be `WithNumVersionsToKeep(int(math.MaxInt32))`

Reply:
In any case, the parameter is an `int` already, so you don't need to manually cast.